lib/Utils/LLVMShorthands.cpp

```diff
@@ -76,10 +106,14 @@ bool matchesSignature(const llvm::Function *F,
       F->getReturnType() == FType->getReturnType()) {
     unsigned Idx = 0;
     for (const auto &Arg : F->args()) {
-      if (Arg.getType() != FType->getParamType(Idx)) {
+      bool TypeMissMatch = ExactMatch ? Arg.getType() != FType->getParamType(Idx) : 
+                                                                  !isTypeMatchForFunctionArgument(FType->getParamType(Idx), Arg.getType());
+      if (TypeMissMatch) {
         return false;
       }
       ++Idx;
+    }
+      }
```

Comment:
These two closing braces are extraneous and cause compilation errors.

I'm trying the github suggestion feature, but I'm not sure how that will work in this case:
```suggestion
```

Reply:
Oh, that was probably my bad and got tangled up with my suggestions. Getting the correct nesting depth and indentation of brackets right with github review suggestions is sometimes hard :laughing:

Reply:
Yes!  The suggestions are a bit tricky since it does not show the surrounding code.